setup.cfg

```diff
@@ -3,11 +3,12 @@
 [metadata]
 name = pyfancy
 description = "Simple terminal formatting in Python"
+long_description = file: README.rst
 # long_description = file: README.MD
 # Versions should comply with PEP440.  For a discussion on single-sourcing
 # the version across setup.py and the project code, see
 # https://packaging.python.org/en/latest/single_source_version.html
-version = 2.4.4
+version = 2.4.5
 # Author details
 author = CosmicWebServices
 author_email = andrewrodebaugh@cosmicsearch.org
```

Comment:
New email is aquahydration@gmail.com

Reply:
@CosmicWebServices , you is Pyfancy author and you can change values for `author` and `author_email` parameters as you want.

Thanks.